index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Coin Toss Solo Build</title>
+</head>
+<link rel="stylesheet" href="styles.css">
+<body>
+    <header><h1>Panda Flip Game</h1></header>
+    <main>
+        <p class= "scoreboard">
+                <span id="wins">Wins: 0</span>
+                <span id="losses">Losses: 0</span>
+        </p>
+        <p>
```

Comment:
I see a lot of people wrap images in p tags and I believe this is because images are inline vs paragraphs are block so it's easier to style them. But really you can have an img tag without a p tag and still style it like a block element if you use the css display property
```
img {
display: block;
}
```